fairlearn/reductions/_exponentiated_gradient/exponentiated_gradient.py

```diff
@@ -211,10 +224,19 @@ def predict(self, X):
     def _pmf_predict(self, X):
         """Probability mass function for the given input data.
 
-        :param X: Feature data
-        :type X: numpy.ndarray or pandas.DataFrame
-        :return: Array of tuples with the probabilities of predicting 0 and 1.
-        :rtype: pandas.DataFrame
+        Calculates predictions for each of the underlying classifiers. The predictions are
```

Comment:
I think we only need to describe "what" and not "how"? And if you want, you could put "how" into Notes--but I don't think that we need to say "how". How about just:
> For each data point, provide the probabilities with which 0 and 1 is returned as a prediction.

I really don't think that you need to provide Notes, but if you do, you can do something like:
> Probabilities are obtained by applying the predictors in `predictors_` and
> then calculating the probability of returning 0 or 1 when one of them is
> chosen according to `weights_`.

Reply:
I agree. If we ever do a more detailed description of Exponentiated Gradient in the user guide or contributor guide we can write this with more detail.